revno: 278
committer: Jelmer Vernooij <jelmer@samba.org>
branch nick: trunk
timestamp: Tue 2008-11-04 14:18:08 +0100
message:
  Fix build.

=== modified file 'Makefile'
--- Makefile	2007-05-09 20:26:21 +0000
+++ Makefile	2008-11-04 13:18:08 +0000
@@ -23,7 +23,7 @@
 	$(CC) $(SHFLAGS) $(CFLAGS) -Wl,--out-implib=ptb.dll.a -o $@ $^
 
 libptb.so.$(VERSION): $(PTBLIB_OBJS:.o=.po)
-	$(CC) $(SHFLAGS) -Wl,-soname,libptb.so.$(SOVERSION) -Wl,$@ $(CFLAGS) -o $@ $^
+	$(CC) $(SHFLAGS) -Wl,-soname,libptb.so.$(SOVERSION) $(CFLAGS) -o $@ $^
 
 libptb.a: $(PTBLIB_OBJS)
 	$(AR) rs $@ $^

